.gitignore

```diff
@@ -22,3 +22,6 @@ website/build/
 
 # for local Docker Compse changes; see https://docs.docker.com/compose/extends/#multiple-compose-files
 docker-compose.override.yml
+
+# enviroment variable files like dotenv
+.env
```

Comment:
See the comment on top of the file

Reply:
If you mean "dotenv" then that is how the file is called. But I can of course also write .env.

Reply:
I mean this comment:

https://github.com/FerretDB/FerretDB/blob/a0cd2e5c5bf56cc51747f399189d924ee31f8197/.gitignore#L1-L2

That line should be in your global `.gitignore` file, not there.

Please also do not resolve conversations after responding:
https://github.com/FerretDB/FerretDB/blob/a0cd2e5c5bf56cc51747f399189d924ee31f8197/.github/PROCESS.md?plain=1#L83-L85

(but please read the whole file)

Reply:
Well, one could add them to dotenv's `.envrc` and have the same problem. But we are not trying to cover all tools contributors could use. Project's `.gitignore` file is for project-specific ignore patterns.